src/NServiceBus.Persistence.AzureStorage/NServiceBus.Persistence.AzureStorage.csproj

```diff
@@ -5,10 +5,10 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <!-- <GeneratePackageOnBuild>true</GeneratePackageOnBuild> -->
```

Comment:
Why is this left here commented out?

Reply:
It was removed. I commented it out instead.

Reply:
Discussed. Reverting.